api/openapi.yaml.in

```diff
@@ -151,6 +151,7 @@ components:
       minLength: 1
       maxLength: 127
       pattern: '^[a-zA-Z0-9](?:[-._a-zA-Z0-9]{0,125}[a-zA-Z0-9])?$'
+      writeOnly: true
```

Comment:
I got this definition for writeOnly.
The writeOnly property is a boolean attribute that can be added to a schema object to indicate that the property is only included in write operations (such as POST or PUT) and should be omitted from read operations (such as GET).

What does making this write-only buy us?
Are you trying to prevent the BootSetName from being returned in the response?

When we designed session templates, we had the thought that there could be multiple boot sets in a single template. Specifying their name helps disambiguate which boot set attributes belongs to which boot set. If I included both 'compute' and 'UAN' boot sets in the same template, how do I know which is which without seeing their names?

Now, in practice, I do not know whether more than one boot set is included in a session template.

Reply:
We chatted about this. The name referenced above is not the one being removed by this PR. This PR does nothing to prevent templates with multiple boot sets.